src/viam/examples/modules/complex/client.cpp

```diff
@@ -56,7 +56,7 @@ int main() {
     }
     bool do_one_ret = gc->do_one("arg1");
     std::cout << "gizmo1 do_one returned: " << do_one_ret;
-    bool do_one_client_stream_ret = gc->do_one_client_stream({"arg1", "arg2", "arg3"});
+    bool do_one_client_stream_ret = gc->do_one_client_stream({"arg1", "arg1", "arg1"});
```

Comment:
I thought `{"arg1", "arg2", "arg3"}` would return true but it actually returns false. Changed to `{"arg1", "arg1", "arg1"}` instead. Kind of an arbitrary decision 😆 .